index.js

```diff
@@ -81,6 +82,49 @@ rbush3d.prototype = {
         return false;
     },
 
+    raycastInv: function (ox, oy, oz, idx, idy, idz, length) {
+        length = length || Infinity;
+        var node = this.data,
+            toBBox = this.toBBox,
+            result,
+            dist = length;
+
+        if (idx === Infinity && idy === Infinity && idz === Infinity) return undefined;
+        if (boxRayIntersects(node, ox, oy, oz, idx, idy, idz) === Infinity) return undefined;
+
+        var nodesToSearch = new Heap(function (a, b) {
```

Comment:
I would use an array instead of constructing a heap like this.  This implementation is likely not very fast.